### PR TITLE
Update job offers in the DevPortal

### PR DESCRIPTION
**Pull request by AliceStatler**

### CODE REVIEW

File: _data/jobs.yml [hunk 1]
```diff
@@ -1,29 +1,29 @@
-- title: Full Stack Developer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/384247
-  description: Be part of a cross-functional team and use your Java and/or TypeScript skills to further develop our new ecommerce platform.
-  location: Hamburg
+- title: Software Developer - 3rd Party Integration, REST API (m/f/d)
```
Comment (dkluever22) [inline _data/jobs.yml:1]:
```suggestion
- title: Software Developer (m/f/d) - 3rd Party Integration, REST API
```
It is done like this for the internship post and I think it's nicer to put the gender information directly after the job title

File: _data/jobs.yml [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+  location: Hamburg/Barcelona
 
-- title: User Experience Designer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/203174
-  description: Keep an overview of our products' UX and develop user-friendly interaction concepts while working closely with our product management team.
-  location: Hamburg
\ No newline at end of file
+- title: Internship Webdesign (m/f/d) - 6 months or 1 year
+  link: https://epages-gmbh.jobs.personio.de/job/359886
+  description: Grown and learn in our Design & Consult team by advising our international customers and creating individual designs.
```
Comment (dkluever22) [inline _data/jobs.yml:28]:
```suggestion
  description: Grow and learn in our Design & Consult team by advising our international customers and creating individual designs.
```

**Review by dkluever22 — APPROVED** [review]
@AliceStatler I cannot say anything about the accuracy of the job IDs but apart from that it looks fine to me. I just have two minor suggestions. Please take a look :)

File: _data/jobs.yml (hunk 1 above)
Comment (AliceStatler) [reply _data/jobs.yml:1]: @dkluever22 But the stuff after the - is part of the job title. It feels a bit odd and this is the actual job title HR put there. I would like to keep it consistent.

Comment (dkluever22) [reply _data/jobs.yml:1]: @AliceStatler Alright. I get that.

File: _data/jobs.yml [hunk 3]
```diff
@@ -1,29 +1,29 @@
...
   description: Be responsible for the further development of our software and support several Scrum teams with your knowledge.
-  location: Hamburg
+  location: Hamburg/Barcelona
 
-- title: User Experience Designer (m/f/d)
-  link: https://epages-gmbh.jobs.personio.de/job/203174
-  description: Keep an overview of our products' UX and develop user-friendly interaction concepts while working closely with our product management team.
-  location: Hamburg
\ No newline at end of file
+- title: Internship Webdesign (m/f/d) - 6 months or 1 year
```
Comment (dkluever22) [inline _data/jobs.yml:26]: @AliceStatler Just to me sure, here "6 months or 1 year" is not part of the Job title, right? Just an additional information.

**Pull request merged**